pom.xml

```diff
@@ -574,6 +584,7 @@ limitations under the License.</inlineHeader>
         <artifactId>formatter-maven-plugin</artifactId>
         <configuration>
           <configFile>eclipse-format.xml</configFile>
+          <lineEnding>LF</lineEnding>
```

Comment:
Good to know. We have tons of Windows build issues in Camel Quarkus and this is certainly one of them.